src/bitte_ci/commander.cr

```diff
@@ -158,7 +158,7 @@ module BitteCI
     def copy_repo
       return unless File.directory?(REPO_ALLOC)
       Git.init
-      repo = Git.clone(REPO_ALLOC, REPO_LOCAL)
+      repo = Git.clone(REPO_ALLOC, REPO_LOCAL, nil, nil)
```

Comment:
@manveru is this valid to indicate we don't care about username / password, here?